{"name": "test_true", "status": "failed", "statusDetails": {"message": "AssertionError: assert False == True\n  +False\n  -True", "trace": "self = <test_leap_year.TestAssert object at 0x00000232C125CF08>\n\n    def test_true(self):\n>       assert is_leap_year(2017) == True\nE       assert False == True\nE         +False\nE         -True\n\ntest_leap_year.py:33: AssertionError"}, "start": 1588051809709, "stop": 1588051809709, "uuid": "6ceddb2e-7c35-4e05-873f-319512de909a", "historyId": "9db7ba8602f272ed967dcdafbb3f3521", "testCaseId": "34603d2208cc04ae1d25f5b0c05a0fa3", "fullName": "test_leap_year.TestAssert#test_true", "labels": [{"name": "suite", "value": "test_leap_year"}, {"name": "subSuite", "value": "TestAssert"}, {"name": "host", "value": "PC-BFD8TY2"}, {"name": "thread", "value": "1676-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_leap_year"}]}